numpy/add_newdocs.py

```diff
@@ -463,6 +463,67 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Create nditers for use in nested loops
+
+    Create a tuple of nditers which iterate in nested loops over different
```

Comment:
can you link this `nditer` with `` `nditer`\ s``?

Reply:
Do you mean literally put into the docstring `` `nditer`\ s``? That then shows up when I do:
```
>>> help(np.nested_iters)
```

Is there an example elsewhere that does what you desire?

Reply:
Yes, I do. I suppose `` `nditer` objects`` would be less objectionable in that sense. That's why I suggest it here and not on the first line.